qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferDbConnector.groovy

```diff
@@ -104,4 +114,78 @@ class OfferDbConnector implements CreateOfferDataSource{
             return generatedKeys[0]
         }
     }
+
+    List<OfferOverview> loadOfferOverview() {
+        List<OfferOverview> offerOverviewList = []
+
+        String query = "SELECT modificationDate, projectTitle, " +
+                "totalPrice, first_name, last_name, email\n" +
+                "FROM offer \n" +
+                "LEFT JOIN person \n" +
+                "ON offer.customerId = person.id"
+
+        Connection connection = connectionProvider.connect()
+        connection.withCloseable {
+            PreparedStatement statement = it.prepareStatement(query)
+            ResultSet resultSet = statement.executeQuery()
+            while (resultSet.next()) {
+                def customer = new Customer.Builder(
+                        resultSet.getString("first_name"),
+                        resultSet.getString("last_name"),
+                        resultSet.getString("email"))
+                        .build()
+                def projectTitle = resultSet.getString("projectTitle")
+                def totalCosts = resultSet.getDouble("totalPrice")
+                def modificationDate = resultSet.getDate("modificationDate")
+                def customerName = "${customer.getFirstName()} ${customer.getLastName()}"
```

Comment:
I'd be more specific here to increase readability
```suggestion
                String projectTitle = resultSet.getString("projectTitle")
                double totalCosts = resultSet.getDouble("totalPrice")
                Date modificationDate = resultSet.getDate("modificationDate")
                String customerName = "${customer.getFirstName()} ${customer.getLastName()}"
```

Reply:
same here. in a local scope, the `def` keyword i would argue is totally fine.